### ISSUE TITLE
NVDA silent on user account control (uac) screens and user account prompt for remote desktop in windows 10

### ISSUE THREAD

**Issue by kieranlittle667** [nvaccess/nvda#9565]
### Steps to reproduce:
Open mstsc and connect to a server, or, try and run CMD as admin with UAC on. (Unsure if issues are related but I suspect they are)

### Actual behavior:
NVDA is silent on these screens
### Expected behavior:
NVDA should read these, presenting the user with either the yes/no / administrator username / password boxes, or in the case of RDP should speak the standard login box.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2019.1.1
#### Windows version:
: Windows 10Ver1803 (64-bit) build 17134.706
#### Name and version of other software in use when reproducing the issue:
n/a
#### Other information about your system:
Lenovo thinkpad e550, 8 GB of ram, SanDisk 128GB ssd
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Have not tested with latest alpha snapshot, however issue was present in earlier versions.

**Comment by DrSooom** [comment]
@kieranlittle667: Three questions:
1. Which synthesizer are you using?
2. Did you copy the current configuration (incl. TTS add-ons) to the system configuration via the General NVDA Settings Dialog?
3. What's the bevahiour after starting Windows at the logon screen?

**Comment by Adriani90** [comment]
If your UAC is set to 34%, NVDA will not report anything on the screen. This is covered already in issue #6236. Closing as duplicate. Please continue the discussion on that issue.

**Comment by lukaszgo1** [comment]
@Adriani90 These two issues don't seems to be related in any way.

**Comment by Adriani90** [comment]
@kieranlittle667 if you switch the value in UAC settings to more than 34 or below, does this solve this issue?

**Comment by Adriani90** [comment]
I am reopening this one, but I think that setting the UAC value to 0 solves this issue. I had similar issues on Windows 8.1 and also Windows 10.

**Comment by kieranlittle667** [comment]
@DrSooom Doesn't matter what synth I use, eSpeak, windows OneCore, etc, issue still present. Yes, I have copied user config to system config, and windows reads the logon screen fine. @Adriani90 I wasn't aware of this issue, but now I've investigated with a higher level of UAC you are correct in that the level set to 34 is the issue. However, i can now see that the issues are not related, as this did not solve my RDP login dialog problems.

**Comment by leonardder** [comment]
@kieranlittle667: Just to make sure, do you have Use NVDA on the Windows logon screen enabled in NVDA's general settings? If so, could you please uncheck the box, press apply, then check it again and save your settings and see whether the problem persists?

**Comment by kieranlittle667** [comment]
@leonardder Yes, I do have that checked. I've unchecked it, applied, checked it again, applied and save config, the issue still exists.

**Comment by amirsol81** [comment]
Hi. The issue is more serious IMHO. That is, even the slider which allows us to alter the User Account Control level isn't read by NVDA. To duplicate, go to the Start Menu, type "User Account" until the item "Change User Account Control settings, Control panel" gets highlighted, and press Enter. Press Tab in the new window to move to the Notification Level slider and press Left/Right to alter it. NVDA remains silent instead of reading the newly focused value like 0, 34, 67, and 100.
This can be duplicated with both NVDA 2020.1 official and NVDA alpha-20204, with and without add-ons, on Windows 10 Pro 64-Bit, Version 2004,.

**Comment by leonardder** [comment]
@amirsol81 The issue you're mentioning, though related to UAC, is not related to this issue itself.

@kieranlittle667 I can reproduce this issue when I disable credential support in an rdp file and thus when starting an RDP session brings me to the logon screen of the remote computer. However, after login and NVDA is started automatically after logon, secure screens read fine.

What happens if you press ctrl+alt+end to bring you to the secure screen that allows you to log off, switch user, etc?

**Comment by amirsol81** [comment]
@leonardder Has the slider issue I mentioned been reported? If not, I'd like to report it as it is important from a usability standpoint.

**Comment by Adriani90** [comment]
yes the slider issue is already there, #9356

**Comment by amirsol81** [comment]
@leonardder Thanks.

**Comment by leonardder** [comment]
Yes. see https://github.com/nvaccess/nvda/issues/9356

**Comment by leonardder** [comment]
I'm sorry @amirsol81 and @Adriani90. After I noticed I wrote a duplicate comment, I tried to remove it, only noticing that my github want mad and removed your comments.

**Comment by Adriani90** [comment]
It seems Narator and Jaws suffer of the same issue with silent screen reader on UAC promt dialog when the value of UAC is set to 34%. Very strange.

**Comment by seanbudd** [comment]
Is this still an issue in Windows 11? Or only Windows 10?

**Comment by Adriani90** [comment]
This is still an issue in Windows 11 with NVDA 2025.2, and also with Narator, when the option for UAC is set to the second last value.

**Comment by Adriani90** [comment]
Note that this does not impact remote desktop only, this impacts any UAC promt appearing when trying to change something (e.g. uninstall something etc.).

**Comment by SaschaCowley** [comment]
Presumably this is an issue because UAC at 34% doesn't switch to the secure desktop for its prompts. Why that is an issue I do not know, but that seems to be the difference.

Users are recommended not to use this setting, so it most likely won't be fixed by NV Access.
